spec/intergration/user_index_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+RSpec.describe User, type: :system do
+  subject { User.new(name: 'Tom', posts_counter: 3, photo: 'https://pic.com', bio: 'Project manager') }
+  before { subject.save }
+  describe 'index page' do
+    it 'I can see the username of all other users.' do
+      visit root_path(subject)
+      expect(page).to have_content(subject.name)
+    end
+    it 'I can see the profile picture for each user' do
+      visit root_path(subject)
+      expect(page).to have_css('img')
```

Comment:
- [ ] Nice job making the required changes, unfortunately, there are a lot of tests still didn't pass. Please fix that.
[Hint] I noticed that all these failed tests because you created a `post_counter` column in the users table and you refer to it as `posts_counter` in the tests.

![image](https://github.com/Roland-Ntwali/Blog-App/assets/68796232/d1411b35-2ad1-4a68-8327-c87069883af8)